demos/sample_azure_iot/sample_azure_iot.c

```diff
@@ -288,7 +289,7 @@ static uint32_t prvSetupNetworkCredentials( NetworkCredentials_t * pxNetworkCred
  * @brief Azure IoT demo task that gets started in the platform specific project.
  *  In this demo task, middleware API's are used to connect to Azure IoT Hub.
  */
-static void prvAzureDemoTask( void * pvParameters )
+void prvAzureDemoTask( void * pvParameters )
```

Comment:
>void [](http://example.com/codeflow?start=0&length=4)

any reason why we removed static?

Reply:
Yes, esp-idf uses a custom function to create FreeRTOS tasks, so they can be pinned to the specific core of the esp32.
Because of that the sample needs to access the task function directly instead of calling `vStartDemoTask`.

Once we agree `prvAzureDemoTask` now must be public, maybe we can agree on a different name?

Reply:
I reverted the sample to use `vStartDemoTask`.